integration-tests/src/test/resources/sitconfig/configoverrides/jms-ClusterJmsSystemResource.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
copyright notice

Reply:
Added.

Reply:
Because of the errors I had to remove the copyright notice in config override files.